src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/SleepInterval.java

```diff
@@ -0,0 +1,33 @@
+package pl.tomaszdziurko.jvm_bloggers.mailing;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.NonNull;
+import lombok.ToString;
+
+import java.util.concurrent.TimeUnit;
+
+@AllArgsConstructor
+@ToString
+@EqualsAndHashCode
+public class SleepInterval {
+
+    public static final SleepInterval NO_SLEEPING =
+        new SleepInterval(TimeUnit.SECONDS, 0);
+
+    final long milliseconds;
+
+    public SleepInterval(@NonNull TimeUnit timeUnit,
+                         long numberOfTimeUnits) {
+        milliseconds = timeUnit.toMillis(numberOfTimeUnits);
+    }
+
+    long asMilliseconds() {
```

Comment:
This method can be removed. We will have `getMilliseconds()` generated by `@Data` if you apply the above.